From 531f2b512d5bee877f91db80ab9d0c1ae41379c6 Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Wed, 20 Mar 2024 09:27:52 +0000
Subject: [PATCH] rdk/grr: klm/cpk/ice_devlink.c: remove devlink_info_driver_name_put()

devlink_info_driver_name_put() was removed in kernel commit 226bf9805506
("net: devlink: let the core report the driver name instead of the drivers").

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/cpk/ice_devlink.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/klm/cpk/ice_devlink.c b/klm/cpk/ice_devlink.c
index 13928c0..4f1b8e6 100644
--- a/klm/cpk/ice_devlink.c
+++ b/klm/cpk/ice_devlink.c
@@ -356,12 +356,6 @@ static int ice_devlink_info_get(struct devlink *devlink,
 		}
 	}
 
-	err = devlink_info_driver_name_put(req, KBUILD_MODNAME);
-	if (err) {
-		NL_SET_ERR_MSG_MOD(extack, "Unable to set driver name");
-		goto out_free_ctx;
-	}
-
 	ice_info_get_dsn(pf, ctx);
 
 	err = devlink_info_serial_number_put(req, ctx->buf);
-- 
2.39.3

